### PR TITLE
Add Do Not Disturb service to Slack

### PR DESCRIPTION
**Pull request by tkdrob**
## Proposed change

Add number platform docs to Slack

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/72291
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/slack.markdown [hunk 1]
```diff
@@ -53,6 +54,12 @@ The added `notify` service will be named after the chat server the app is instal
 
 {% include integrations/config_flow.md %}
 
+## Integration Entities
+
+One number entity will be created:
+
+- **Do Not Disturb**: Set Do Not Disturb period. This starts immediately when set and decreases until depleted.
```
Comment (frenck) [inline source/_integrations/slack.markdown]: Number entities are not allowed to decrease automatically (they are not countdown timers).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/slack.markdown [hunk 2]
```diff
@@ -53,6 +54,12 @@ The added `notify` service will be named after the chat server the app is instal
 
 {% include integrations/config_flow.md %}
 
+## Integration Entities
+
+One number entity will be created:
+
+- **Do Not Disturb Period**: Set Do Not Disturb period. This starts immediately when set.
```
Comment (veleek) [inline source/_integrations/slack.markdown]: This should indicate the unit of time.  Is it minutes, seconds, etc?

**Comment by tkdrob** [comment]
I'm not so sure we need this with the current state of the parent.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**